printer_mods/hbo/Voron0_Skirts/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Collection of Voron 0 Skirts
+
+
+
+
+This is a Collection of Voron 0 skirts that provide some additional features and bring the asthetics closer to the bigger VORON brothers.
+
+![Printed Panel](Images/VORON_V0.1_2022.png)
+
+Right now there are 3 variations for the and one commmon rear.
+To start with the rear ..
+
+## Rear Skirts
+
+The rear skirts match all the different front skirts and have the following features
+
+- integrated power inlet mount. To be used with the "standard" v0 power inlet
+- 2 keystone mount to get acces to the raspery ports
+- bowden clip mount support. Disigned for the E3D style bowden clips. This allows to push the fillament and not the tube from the outside. Read for the rabit feader :)
+
+
+![Printed Panel](Images/VORON_V0.1_Rear.png)
+
+## Headless
+
+This version is intendet for users that like to run the V0 in headless mode i.e. without display
+
+
+![Printed Panel](Images/VORON_V0.1_Headless.png)
+
+## Mini Display
+
+This version is desiged for the 12864 mini display (same as the V2.4 display)
+The design includes a little part to get acces to the reset switsch.
+
+![Printed Panel](Images/VORON_V0.1_MiniDisplay.png)
+
+## Waveshare 4.3" Display
+
+This version is desiged for the Waveshare 4.3inch Capacitive Touch Display for Raspberry Pi, DSI Interface
+The mounts are desihed to bolt on the frame and display with original screws from the back.
+Therfore similar 4.3 inch display might not work without modifications
+I'm using a 60cm DSI cable to connect the diplay with a Raspery back on the back of the V0 
+(a Raspberry Pico mount mod is in preparation)
+
+### Reset / Emergency switch
+The fron display mount also provide a mounting option for a reset or inmergency switch.
+I'm using this switch to reset a BTT PSU relays. 
+
+The switch in the cad is a standard 7mm momentray switch that is easy to source even from local providers.
+Make shire to get a Normal OPEN switch if you plan to use it with the BTT PSU Relays.
+(A Relays mount and firmware configs are in preparation)
+
+![Printed Panel](Images/VORON_V0.1_Waveshare.png)
+
+## Printing
+
+All parts print with standard Voron print setting without any supports.
+All parts fit on a V0. However the max out the build volume on the z axis.
+If you need to print on a V0 make shure that you bed springs are really stiff by adding more tention with washers etc.
+I had some problems getting nice parts out of the V0 due to the nozzle mooving he part dur to flex on the bed springs.
```

Comment:
`mooving` -> `moving`
`he` -> `the`
`dur` -> `due`

Reply:
fixed